docs/user-manual.md

```diff
@@ -21,7 +21,7 @@ To stop the pie, follow these steps:
    $ NAMESPACE=<Set the namespace of the pie.>
    $ kubectl -n ${NAMESPACE} scale deployments --replicas=0 pie
    ```
-2. Wait for Pods to stop.
```

Comment:
What does the "pie Pod" mean? 

If you mean the controller pod, I think "the controller Pod" is better.
(Though the default name of the controller pod is "pie-XYZ...", [it can be changed](https://github.com/topolvm/pie/blob/b552af6461f882c09fb58d20098a2b6767639819/charts/pie/templates/deployment.yaml#L4).

If you mean all the pie-related pods, "the pie Pod**s**" is correct.

Reply:
Thanks for your review. This should be "the controller Pod." I'll fix this.

Reply:
Updated. [compare](https://github.com/topolvm/pie/compare/f21bbedd9a43503ee9eda8507a1dbc575e179c50..1cb76dc95662cf74e29fd064f8c6fd32b544cea2)